create-kafka-queues/create-topics.sh

```diff
@@ -51,6 +51,7 @@ if [ $# -eq 1 ]; then
 else
   printenv
 
+  printf 'Staring create-topics at %s \n' "$(date)"
```

Comment:
Why are you staring? 😆 
```suggestion
  printf 'Starting create-topics at %s \n' "$(date)"
```